### PR TITLE
XDA-33 Alarm Table Sorting

### PR DESCRIPTION
**Pull request by collins-self**
Sorting for the tables at the end of the alarm wizard has been fixed.

### CODE REVIEW

File: Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx [hunk 1]
```diff
@@ -234,71 +232,118 @@ const WizardTest = () => {
...
+                                    TbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: 400, width: '100%' }}
+                                    Data={channelList}
+                                    SortKey={sort}
+                                    Ascending={asc}
+                                    OnSort={(d) => {
                                         if (sort === d.colKey)
                                             setAsc(!asc)
                                         else
                                             setSort(d.colField)
                                     }}
```
Comment (gcsantos-gpa) [inline Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx:254]: This was the only table changed with a sort function. What changed to make this work over the previous version?

**Review by gcsantos-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx [hunk 2]
```diff
@@ -234,71 +232,118 @@ const WizardTest = () => {
...
+                                    Selected={(item) => false}
+                                    KeySelector={(key) => key.NumberRaised}
+                                >
+                                    <ReactTable.Column
+                                        Key='Severity'
+                                        Field={'Severity'}
+                                        Content={({ item }) => {
+                                            return <p style={{ color: severities.find(s => s.Name == item.Severity).Color }}>{item.Severity}</p>
+                                        }}
+                                        AllowSort={true}
```
Comment (elwills) [inline Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx:319]: Would it not be best to set the `AllowSort` flag to `false` if we're preventing sorting?

Comment (gcsantos-gpa) [reply Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx:319]: I think that would be wise.

Comment (gcsantos-gpa) [reply Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx]: Rebased to #360 and retargeted PR

**Review by gcsantos-gpa — APPROVED** [review]

**Pull request merged**